### PR TITLE
Latex $$ fix

### LINKED ISSUE #306: [BUG] latex() assumes $$ in LaTeXString
**Describe the bug**

`get_latex_svg()` assumes the LaTeXString passed is bracketed by `$$`. However, not all LaTeXString constructors auto append the `$$`. From the LaTeXString [README](https://github.com/stevengj/LaTeXStrings.jl):

> Finally,` you can use the lowest-level constructor LaTeXString(s). The only advantage of this is that it does not automatically put $ at the beginning and end of the string.

**To Reproduce**

1. Julia Version (i.e. output of `julia -v`):
v1.5.3
2. Operating system (Mac, Linux, Windows):
Mac & Linux
3. Javis version (i.e output of `] status Javis` in the REPL)
v0.3.3
4. Minimum working code example that led to bug:

```
using Javis, LaTeXStrings
Javis.get_latex_svg(LaTeXString(L"\sqrt{x}") == Javis.get_latex_svg(LaTeXString("\\sqrt{x}")
```

### PR DESCRIPTION
**Pull request by agerlach**
## PR Checklist

If you are contributing to `Javis.jl`, please make sure you are able to check off each item on this list:

- [x] Did I update `CHANGELOG.md` with whatever changes/features I added with this PR?
- [x] Did I make sure to only change the part of the file where I introduced a new change/feature?
- [x] Did I cover all corner cases to be close to 100% test coverage (if applicable)?
- [x] Did I properly add Javis dependencies to the `Project.toml` + set an upper bound of the dependency (if applicable)?
- [x] Did I properly add test dependencies to the `test` directory (if applicable)?
- [x] Did I check relevant tutorials that may be affected by changes in this PR?
- [x] Did I clearly articulate why this PR was made the way it was and how it was made?

**Link to relevant issue(s)**
Closes #306

**How did you address these issues with this PR? What methods did you use?**
Check if `$` in the LaTeXString before removing.

### CODE REVIEW

**Comment by Wikunia** [comment]
Thanks a lot! Do you mind to change the version directly in the Project.toml? :wink:

**Comment by Wikunia** [comment]
Regarding the test failure:
You can only use LaTeX commands which are in the `LaTeXSVG` dict

**Comment by agerlach** [comment]
> Regarding the test failure:
> You can only use LaTeX commands which are in the `LaTeXSVG` dict

This doesn't make sense to me by looking at the source. Tests pass locally. The CI logs seem to indicate that `tex2svg` is not installed.

**Comment by agerlach** [comment]
Are you saying it needs to be included in the `Dict` constructor in `latexsvgfile.jl`? If, so it will end up just bypassing the code that needs tested. I can understand why in your other tests you want to bypass `txt2svg` and use a reference svg, but does that makes sense here?

**Comment by Wikunia** [comment]
Good point we might want to save the `ts` in the dict and not the text, right? Then you can do your step before checking if it is cached. It currently fails because it's not cached and then we try to render it but haven't installed the npm module on GitHub.

**Comment by agerlach** [comment]
That would work, but will require changing the types used in LaTeXSVG as the key won't be `LaTeXString` anymore. I'm unclear on what the upstream affects of that would be. 

Two other thoughts:

1. Why not just install the npm module?  
2. I could bypass all this by moving to code to a function, e.g `strip_eq(txt::LaTeXString) = txt[1] == '$' ? txt[2:(end-1)] : txt`. The test will then just be a test of this function.

**Comment by Wikunia** [comment]
I like the first option but I have not enough knowledge how to do it. I'm fine with option 2 for now

**Comment by agerlach** [comment]
OK, I will do 2. However, as a future note, from a cache perspective it would be better to cache based on the stripped string because now you can have two different keys that produce the same svg., e.g. `\\sqrt{x}` and `$\\sqrt{x}$`.

**Comment by Wikunia** [comment]
Totally agree. Maybe add a `Todo:` in the code 😊

**Comment by agerlach** [comment]
done

File: src/latex.jl [hunk 1]
```diff
@@ -86,13 +86,21 @@ function animate_latex(text, pos::Point, t, object)
     translate(-pos)
 end
```
Comment (Wikunia) [inline src/latex.jl:88]: If you don't mind a short docstring would be great :smile:

Comment (agerlach) [reply src/latex.jl:88]: done

**Comment by Wikunia** [comment]
Thanks a lot! Enjoy the holidays :wink:

**Pull request merged**